packages/near-snap/README.md

```diff
@@ -0,0 +1,62 @@
+# @near-wallet-selector/near-snap
+[Metamask Snap](https://metamask.io/snaps/) system that allows anyone to safely expand the capabilities of MetaMask. Currently is pre-release software running on [Metamask Flask](https://metamask.io/flask/).
+
+This package implement NEAR snap for NEAR Wallet Selector.
+
+## Installation and Usage
+
+The easiest way to use this package is to install it from the NPM registry, this package requires `near-api-js` v0.44.2 or above:
+
+```bash
+# Using Yarn
+yarn add near-api-js@^0.44.2
+
+# Using NPM.
+npm install near-api-js@^0.44.2
```

Comment:
Can you remove the `@^0.44.2` part from the installation section and just leave it like this:

```bash
# Using Yarn
yarn add near-api-js

# Using NPM.
npm install near-api-js
```

We have changed the range and this way it will mean that the latest version of near-api-js will be installed in a dApp.